Cargo.toml

```diff
@@ -1,19 +1,20 @@
 [package]
 name = "yaiwr"
-version = "0.0.1"
+version = "0.0.2"
 authors = ["Pavel Durov"]
 edition = "2021"
+rust-version = "1.67.0"
 
 [[bin]]
 doc = false
 name = "yaiwr"
 
 [build-dependencies]
 cfgrammar = "0.13"
-lrlex = "0.13"
-lrpar = "0.13"
+lrlex = "0.13.1"
```

Comment:
I think I needed that to use the .l file change :thinking:

Reply:
I dunno, you look pretty professional there, with your science experiment :rofl: 

Being serious a moment...

According to [semantic versioning](https://semver.org/) (which Rust crates should use by convention) version `0.13` and `0.13.1` should be functionally the same. That update shouldn't (if @ltratt got his semver right) shouldn't unlock any new functionality.

Reply:
yeah 0.13.1 is a patch so shouldn't unlock new functionality
I tried again and it seems to work with both versions (maybe I did something weird last time I tried)
I will leave the new version change, just cause it's newer

Reply:
>  version 0.13 and 0.13.1 should be functionally the same. That update shouldn't (if @ltratt got his semver right) shouldn't unlock any new functionality.

That's not what semver means :) 0.13 -> 0.13.1 means "we didn't break anything that existed in the API but we might add new stuff". In this case "new stuff" includes "we now accept things we used to reject" (see https://github.com/softdevteam/grmtools/blob/master/CHANGES.md).

Reply:
That's not how I interpret [the docs](https://semver.org/):

```
Given a version number MAJOR.MINOR.PATCH, increment the:

    MAJOR version when you make incompatible API changes
    MINOR version when you add functionality in a backwards compatible manner
    PATCH version when you make backwards compatible bug fixes
```

With 0.13(.0) -> 0.13.1, we are bumping `PATCH`, so it should contain only bugfixes and no new functionality, right?

Reply:
@ltratt still curious about this.

Reply:
Remember that semver is sort-of broken for 0.x (i.e. "less than MAJOR=1") and you can't use the same rules to interpret as for 1.x. Happy to discuss this further, but let's do it outside of this PR.